### PR TITLE
ReemHany_LCDWithOS

### PR DESCRIPTION
**Pull request by ReemHany136**

### CODE REVIEW

File: Reem Hany/LCD_UnitTesting/LCDTesting.c [hunk 1]
```diff
@@ -0,0 +1,298 @@
...
+
+    }
+
+    /*
+     * Initialized Module
+     * Expected Output is E_OK
+    */
+    /***********************************/
+
+    ERR = LCD_ExecuteCommand(0x00);
```
Comment (YehiaSprints) [inline Reem Hany/LCD_UnitTesting/LCDTesting.c]: you need to call the LCD init function before this line.

File: Reem Hany/LCD_UnitTesting/LCDTesting.c [hunk 2]
```diff
@@ -0,0 +1,298 @@
...
+    ERROR_STATUS ERR;
+    /*
+     * Uninitialized Module
+     * Expected Output is LCD_ERROR + E_MULTIPLE_INITIALIZION
+    */
+    /***********************************/
+
+    ERR = LCD_ExecuteCommand(0x00);
+
+    if(ERR == LCD_ERROR + E_UNINITIALIZED_MODULE)
```
Comment (YehiaSprints) [inline Reem Hany/LCD_UnitTesting/LCDTesting.c]: What if the command is wrong?

File: Reem Hany/LCD_UnitTesting/LCDTesting.c [hunk 3]
```diff
@@ -0,0 +1,298 @@
...
+
+    }
+
+    /*
+     * Initialized Module
+     * Expected Output is E_OK
+    */
+    /***********************************/
+    LCD_Init();
+    ERR = LCD_WriteData(0x00);
```
Comment (YehiaSprints) [inline Reem Hany/LCD_UnitTesting/LCDTesting.c]: what if the data or characters cannot be displayed on the LCD like the Arabic characters?

**Comment by YehiaSprints** [comment]
Rename the pull request(your nickName_challenge name)

**Comment by YehiaSprints** [comment]
No Comments from Mohammed Tarek.

**Comment by BassamSharkawy-Sprints** [comment]
Please apply Yahia's/M.Tarek's reviews and merge.

**Pull request closed without merging**